plugins/flytekit-ray/flytekitplugins/ray/task.py

```diff
@@ -65,7 +64,7 @@ def __init__(self, task_config: RayJobConfig, task_function: Callable, **kwargs)
         self._task_config = task_config
 
     def pre_execute(self, user_params: ExecutionParameters) -> ExecutionParameters:
-        init_params = {"address": self._task_config.address}
```

Comment:
thanks. I think this only works for local execution. For those people who still want to use it, they can use `RAY_ADDRESS` env.